packages/helpers/src/hooks/useIsSmallScreen.ts

```diff
@@ -1,22 +1,39 @@
-import { useLayoutEffect, useState } from "react";
-import debounce from "lodash/debounce";
+import { useEffect, useState } from "react";
 
-const useIsSmallScreen = (threshold = 768, wait = 250) => {
-  const [isSmallScreen, setSmallScreen] = useState(
-    window.innerWidth < threshold,
+export type Breakpoint = "xs" | "sm" | "md" | "lg";
+
+const breakpoints: Record<Breakpoint, string> = {
+  xs: "48rem",
+  sm: "67.5rem",
+  md: "80rem",
+  lg: "100rem",
+};
+
+export function useIsSmallScreen(threshold: Breakpoint): boolean;
```

Comment:
Why is the function `useIsSmallScreen` declared three times here and does it need to be exported?